mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -248,14 +251,14 @@ public void logout(MailboxSession session) {
      */
     protected StoreMessageManager createMessageManager(Mailbox mailbox, MailboxSession session) throws MailboxException {
         return new StoreMessageManager(DEFAULT_NO_MESSAGE_CAPABILITIES, getMapperFactory(), getMessageSearchIndex(), getEventBus(),
-                getLocker(), mailbox, quotaManager,
+            getLocker(), mailbox, quotaManager,
             getQuotaComponents().getQuotaRootResolver(), configuration.getBatchSizes(),
             getStoreRightManager(), preDeletionHooks, new MessageStorer.WithoutAttachment(mailboxSessionMapperFactory, messageIdFactory, new MessageFactory.StoreMessageFactory()));
     }
 
     @Override
     public MessageManager getMailbox(MailboxPath mailboxPath, MailboxSession session)
-            throws MailboxException {
+        throws MailboxException {
```

Comment:
Please keep previous indent

Reply:
happened by auto format code.

Reply:
That's not a reason ^^

Reply:
those change not from me, it was long ago by the history, but i will remake them to normal.